datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -359,6 +359,8 @@ def parse_metric_family(self, response, scraper_config):
         :param response: requests.Response
         :return: core.Metric
         """
+        if response.encoding is None:
```

Comment:
We should also add `accept: text/plain` to `get_http_handler` at line 344 above too.

Reply:
Sure, done. Also added a test.